README.md

```diff
@@ -655,28 +655,192 @@ _You can enable the following settings in Xcode by running [this script](resourc
 
   </details>
 
-* <a id='attributes-on-prev-line'></a>(<a href='#attributes-on-prev-line'>link</a>) **Place function/type attributes on the line above the declaration**. [![SwiftFormat: wrapAttributes](https://img.shields.io/badge/SwiftFormat-wrapAttributes-7B0051.svg)](https://github.com/nicklockwood/SwiftFormat/blob/master/Rules.md#wrapAttributes)
+* <a id='attributes-on-prev-line'></a>(<a href='#attributes-on-prev-line'>link</a>) **Place attributes for functions, types, and computed properties on the line above the declaration**. [![SwiftFormat: wrapAttributes](https://img.shields.io/badge/SwiftFormat-wrapAttributes-7B0051.svg)](https://github.com/nicklockwood/SwiftFormat/blob/master/Rules.md#wrapAttributes)
 
   <details>
 
   ```swift
   // WRONG
   @objc class Spaceship {
 
+    @ViewBuilder var controlPanel: some View {
+      // ...
+    }
+
     @discardableResult func fly() -> Bool {
+      // ...
     }
+
   }
 
   // RIGHT
   @objc
   class Spaceship {
 
+    @ViewBuilder
+    var controlPanel: some View {
+      // ...
+    }
+
     @discardableResult
     func fly() -> Bool {
+      // ...
     }
+
+  }
+  ```
+
+  </details>
+
+* <a id='stored-property-attributes-on-same-line'></a>(<a href='#stored-property-attributes-on-same-line'>link</a>) **Place simple attributes for stored properties on the same line as the rest of the declaration**. Complex attributes with named arguments, or more than one unnamed argument, should be placed on the previous line. [![SwiftFormat: wrapAttributes](https://img.shields.io/badge/SwiftFormat-wrapAttributes-7B0051.svg)](https://github.com/nicklockwood/SwiftFormat/blob/master/Rules.md#wrapAttributes)
+
+  <details>
+
+  ```swift
+  // WRONG. These simple property wrappers should be written on the same line as the declaration. 
+  struct SpaceshipDashboardView {
+
+    @State
+    private var warpDriveEnabled: Bool
+
+    @ObservedObject
+    private var lifeSupportService: LifeSupportService
+
+    @Environment(\.controlPanelStyle) 
+    private var controlPanelStyle
+
+  }
+
+  // RIGHT
+  struct SpaceshipDashboardView {
+
+    @State private var warpDriveEnabled: Bool
+
+    @ObservedObject private var lifeSupportService: LifeSupportService
+
+    @Environment(\.controlPanelStyle) private var controlPanelStyle
+
+  }
+  ```
+
+  ```swift
+  // WRONG. These complex attached macros should be written on the previous line.
+  struct SolarSystemView {
+
+    @Query(sort: \.distance) var allPlanets: Planet
```

Comment:
I get a build error locally if I don't fully qualify the key path but I'm OK keeping what we have as-is since it matches Apple documentation. There could be something that I'm missing in my code example.

<img width="2036" alt="Screenshot 2023-12-12 at 4 04 36 PM" src="https://github.com/airbnb/swift/assets/1791049/e3c3c3b0-585c-4e13-a0b0-a6cb9adde963">